### PR TITLE
Add D5Next Pump from AquaComputer to supported devices

### LINKED ISSUE #357: D5Next Missing
AquaComputer released a new waterpump called D5 Next. it behaves the same like the aquastreamXT, which is already in the library. I tried to setup the project in VSto add the stuff myself, but wasn't successful yet. really complicated to setup a .NET development environment. maybe one can share some infos here?
Nevermind, i got it running now. wil try to add the pump. let's see

### PR DESCRIPTION
**Pull request by eikaramba**
This fixes #357 . Took me some time but it now shows the water temperature and pump rpm in the monitor.
grafik

Please merge this so that it can be integrated into https://github.com/Rem0o/FanControl.Releases to control the fans based on the water temperature :)

### CODE REVIEW

File: LibreHardwareMonitorLib/Hardware/Controller/AquaComputer/AquaComputerGroup.cs [hunk 1]
```diff
@@ -26,6 +26,16 @@ public AquaComputerGroup(ISettings settings)
...
+                    case 0xF00E:
+                        {
+                            var device = new D5Next(dev, settings);
+                            _report.AppendLine($"Device name: {productName}");
+                            _report.AppendLine($"Firmware version: {device.FirmwareVersion}");
+                            _report.AppendLine();
+                            _hardware.Add(device);
+
+                            break;
+                        }
```
Comment (PhyxionNL) [inline LibreHardwareMonitorLib/Hardware/Controller/AquaComputer/AquaComputerGroup.cs:38]: Can you reformat this block?

File: LibreHardwareMonitorLib/Hardware/Controller/AquaComputer/D5Next.cs [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+// All Rights Reserved.
+
+using System;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using HidSharp;
+
+namespace LibreHardwareMonitor.Hardware.Controller.AquaComputer
+{
+
```
Comment (PhyxionNL) [inline LibreHardwareMonitorLib/Hardware/Controller/AquaComputer/D5Next.cs:13]: This must also be reformatted.

File: LibreHardwareMonitorLib/Hardware/Controller/AquaComputer/D5Next.cs [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            base.Close();
+        }
+
+        public override void Update()
+        {
+            //Reading output report instead of feature report, as the measurements are in the output report
+            _stream.Read(_rawData);
+
+
```
Comment (PhyxionNL) [inline LibreHardwareMonitorLib/Hardware/Controller/AquaComputer/D5Next.cs:72]: Lots of new lines here ;)

**Review by PhyxionNL — CHANGES_REQUESTED** [review]
Some trivial stuff, rest looks good to merge.

**Comment by eikaramba** [comment]
I fixed the formatting (hopefully). Let me know if something is still wrong, unfortunately i never used Visual Studio only VSCode and don't know how to format automatically according to some formating style. I did the changes manually now.
Thanks for the quick reply

**Pull request merged**